recommender/engine/rl_agent/models/history_embedder.py

```diff
@@ -1,30 +1,46 @@
+# pylint: disable=missing-module-docstring
+
 import torch
+from torch import nn
 
 
 class HistoryEmbedder(torch.nn.Module):
     """
-    Model used for transforming services history (list of services tensors
-     in temporal order) into history tensor. It should be used and trained inside actor and critic.
+    Model used for transforming services history (list of service tensors
+     in temporal order) into a history tensor.
+     It should be used and trained inside both actor and critic.
     """
 
-    def __init__(self):
-        super().__init__()
-        self.rnn = None
+    def __init__(self, SE: int, num_layers: int = 3, dropout: float = 0.5):
+        """
+        NOTE: hidden_size is always equal to SE
 
-        # TODO: implement the rest of the initialization (layers)
-        pass
+        Args:
+            SE: service embedding dimension
+            num_layers: number of hidden layers
+            dropout: a dropout probability
+        """
+        super().__init__()
+        self.rnn = nn.RNN(
```

Comment:
Use GRU or LSTM, because input tensors will have rather big sequence lenght and simple RNN can't handle vanishing gradients in this case.